client/components/command-palette/index.tsx

```diff
@@ -239,13 +241,15 @@ const CommandPalette = () => {
 	const [ placeHolderOverride, setPlaceholderOverride ] = useState( '' );
 	const [ search, setSearch ] = useState( '' );
 	const [ selectedCommandName, setSelectedCommandName ] = useState( '' );
-	const [ isOpen, setIsOpen ] = useState( false );
+	const [ isOpenLocal, setIsOpenLocal ] = useState( false );
+	const isOpenGlobal = useSelector( ( state: object ) => isCommandPaletteOpen( state ) );
```

Comment:
Would you be open to avoid using a Redux selector here? @Automattic/t-rex is working on extracting the component palette outside of Calypso so it can run on WP Admin too.

Unfortunately, the Calypso global Redux state is not available in WP Admin, so we're removing any usage of `useSelector`. See paYJgx-4Cq-p2 and https://github.com/Automattic/wp-calypso/pull/87422.

An alternative way to implement this that will also work with the decoupled command palette is by injecting a new `isOpen` property to the component that is set externally by the consumer (either Calypso or WP Admin).

So [this](https://github.com/Automattic/wp-calypso/blob/5577003cf2e730f8f0ae12193ddf5b5336764a29/client/layout/index.jsx#L340) would change to:

```
const isOpenGlobal = useSelector( ( state: object ) => isCommandPaletteOpen( state ) );

// ...
<AsyncLoad require="calypso/components/command-palette" placeholder={ null } isOpen={ isOpenGlobal }/>
```

Reply:
Thank you for letting me know, @mmtr 🙂 

>An alternative way to implement this that will also work with the decoupled command palette is by injecting a new isOpen property to the component that is set externally by the consumer (either Calypso or WP Admin).

I will be removing the Redux-related code from this component! Extracting it to the package will be left to you at https://github.com/Automattic/wp-calypso/pull/87422.

Reply:
https://github.com/Automattic/wp-calypso/pull/87628/commits/0c33def1e1637f109a165f65322dcb5a7168a4b0